package.json

```diff
@@ -20,9 +20,21 @@
     "changelog": "conventional-changelog -p videojs -i CHANGELOG.md -s",
     "build": "grunt dist",
     "minify": "babel-node build/minify.js",
+    "cssmin": "npm-run-all cssmin:*",
+    "cssmin:cdn": "cleancss dist/alt/video-js-cdn.css -o dist/alt/video-js-cdn.min.css",
+    "cssmin:default": "cleancss dist/video-js.css -o dist/video-js.min.css",
+    "sass": "npm-run-all sass:* autoprefixer",
+    "sass:cdn": "node-sass src/css/vjs-cdn.scss dist/alt/video-js-cdn.css",
+    "sass:default": "node-sass src/css/vjs.scss dist/video-js.css",
```

Comment:
Tests still rely on `build/temp/video-js.css`, we should update that and also make sure that there aren't any other uses of `build/temp/video-js.css` like in sandbox or doc files.

Reply:
good call